public/js/graph/sidebar/sidebar_events.js

```diff
@@ -124,3 +125,19 @@ function enableReset() {
 
     fillFCECount();
 }
+
+/**
+ * Enables the Focuses nav in the sidebar if the CS graph is selected.
+ * @param:{string} id ID of the graph we just selected
+**/
+function changeFocusEnable(id) {
+    var currentGraph = graphs[id - 1].title;
+    if (currentGraph.indexOf("Computer Science") >= 0)
+    {
```

Comment:
Not a problem. Watch out though, it looks like the `else` statement walked onto the next line.